src/app/containers/Metadata/index.test.jsx

```diff
@@ -67,6 +74,8 @@ const linkedDataProps = (createdBy, logoUrl, optimoId, seoHeadline) => ({
 const dotComOrigin = 'https://www.bbc.com';
 const dotCoDotUKOrigin = 'https://www.bbc.co.uk';
 
+process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH = 'http://foo.com';
```

Comment:
Curious how the snapshots have come out as `https` when this line is `http`...?

Reply:
That'll be because I was inconsistent here by using `https`. Will update to keep consistent 👍 

https://github.com/bbc/simorgh/blob/3c6c84f58422f5f1b607d1ec498d167110ca3701/src/app/components/Metadata/index.test.jsx#L67